code/API_definitions/device-status.yaml

```diff
@@ -664,9 +672,9 @@ components:
         device:
           $ref: "#/components/schemas/Device"
         type:
-          $ref: "#/components/schemas/SubscriptionCreationEventType"
+          $ref: "#/components/schemas/SubscriptionEventType"
 
-    SubscriptionEventType:
+    NotificationEventType:
```

Comment:
Shouldn't the NotificationEventType extend the SubscriptionEventType, to avoid duplications. Or are there cases where we subscribe to an event, which is not triggered?

Reply:
As discussed during the meeting, I'm dubious that we can extend an enum with the `allOf` and furthermore we will have an issue with the description so probably we can keep as it.